src/Wrapper/Matrix.cs

```diff
@@ -40,7 +40,15 @@ namespace ArrayFire
 
 	public static class Matrix
 	{
-		[MethodImpl(MethodImplOptions.AggressiveInlining)]
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static Array MatMul(Array lhs, Array rhs, bool lconj = false, bool rconj = false)
+        {
+            IntPtr ptr;
+            Internal.VERIFY(AFBlas.af_matmul(out ptr, lhs._ptr, rhs._ptr, lconj ? af_mat_prop.AF_MAT_CONJ : af_mat_prop.AF_MAT_NONE, rconj ? af_mat_prop.AF_MAT_CONJ : af_mat_prop.AF_MAT_NONE));
```

Comment:
what if a user wants to transpose `lhs` ? mat prop can take [many values](http://arrayfire.org/docs/defines_8h.htm#a67fd2718ccb6233ec081fc7b4757421d). I don't it would be wise add new functions different variations of mat_prop values.

Reply:
I don't see any change in this function signature or body ?

Reply:
Sorry, don't understand it.

Reply:
The current signature of `MatMul` uses two boolean values to indicate whether inputs are conjugated. However, they don't provide the ability to tranpose the inputs like the C-API or any other wrapper we have now.
See [python-wrapper](http://arrayfire.org/arrayfire-python/arrayfire.blas.html?highlight=matmul#arrayfire.blas.matmul) and [rust-wrapper](http://arrayfire.org/arrayfire-rust/arrayfire/fn.matmul.html) matmul API for examples of what I am referring to.